base/error.jl

```diff
@@ -78,7 +78,29 @@ systemerror(p, b::Bool; extrainfo=nothing) = b ? throw(Main.Base.SystemError(str
 
 ## assertion functions and macros ##
 
+
+"""
+    assert(cond)
+
+Throw an [`AssertionError`](@ref) if `cond` is `false`.
+Also available as the macro `@assert expr`.
```

Comment:
While you are at it, perhaps change to ``Also available as the macro [`@assert`](@ref).``?